Makefile

```diff
@@ -12,16 +12,22 @@ zip_dir = $(name)/
 
 .PHONY: test
 
-
 package: zip
 
-test:
+test: unittest
 	@echo -e "\e[1;37m=\e[1;34m Starting tests\e[0m"
 	tox -e py27,py36
 	pylint $(name)/*.py
 	pylint $(name)/resources/lib/*/*.py
 	@echo -e "\e[1;37m=\e[1;34m Tests finished successfully.\e[0m"
 
+unittest:
+	@echo -e "\e[1;37m=\e[1;34m Starting unit tests\e[0m"
+	PYTHONPATH=$(name) python $(name)/vrtnutests/apihelpertests.py
+	PYTHONPATH=$(name) python $(name)/vrtnutests/urltostreamservicetests.py
```

Comment:
I think its better to remove the urltostreamservicetests since they will only work property when logged in with an account.

Reply:
All three work fine on my system just like that ?

Reply:
Aha, I am not really testing anything. So like I asked you before, how should I invoke these tests ?
These are only classes.